src/script/main/app.ts

```diff
@@ -138,6 +145,25 @@ function doRedirect(signOutReason) {
 }
 
 class App {
+  apiClient: APIClient;
+  backendClient: BackendClient;
+  logger: any;
```

Comment:
Please import `Logger` and use it here.

Reply:
done in https://github.com/wireapp/wire-webapp/pull/9511/commits/f89353a3a5793eaa4bf6a95c1cd629ec63828ee1